data.tf

```diff
@@ -7,8 +7,13 @@ data "cloudfoundry_user" "user" {
   org_id = data.cloudfoundry_org.org.id
 }
 
+data "hsdp_config" "cf" {
+  region = var.cf_region
```

Comment:
IIRC this is not needed if the provider is correctly configured with the region. (which it should be anyway)

Not sure this is worth the additional setting here.

Reply:
Good point, will remove.